README.md

```diff
@@ -76,7 +76,7 @@ sudo pacman -S --noconfirm --needed eigen3 hdf5 opencv protobuf vtk yay python2-
 
 #### Gstreamer:
 ```bash
-sudo pacman -S --needed $(pacman -Ssq gstreamer)
+sudo pacman -S --needed gstreamer gst-plugins-bad gst-plugins-base gst-plugins-base-libs gst-plugins-good gst-plugins-ugly
```

Comment:
Are you sure this is applicable to Arch Linux the same way?

Reply:
No, it doesn't cause a conflict. However 32 packages are installed, with only 6 being required.

These are the 32:
```
clutter-gst
gnome-video-effects
gst-libav
gst-plugins-bad
gst-plugins-base
gst-plugins-base-libs
gst-plugins-good
gst-plugins-ugly
gst-python
gst-python2
gstreamer
gstreamer-vaapi
libcanberra-gstreamer
manjaro-gstreamer
phonon-qt5-gstreamer
qt-gstreamer
totem
gambas3-gb-media
gst-editing-services
gst-plugins-espeak
gst-transcoder
gstreamermm
gstreamermm-docs
parole
parole-gtk3
pitivi
transmageddon
lib32-gst-plugins-base
lib32-gst-plugins-base-libs
lib32-gst-plugins-good
lib32-gstreamer
lib32-libcanberra-gstreamer
```

Packages like ```gnome-video-effects``` [link](https://gitlab.gnome.org/GNOME/gnome-video-effects) and ```pitivi``` [link](http://www.pitivi.org/) have nothing to do with gstreamer and are just bloating up the target destination.